demoapp/testimony_summary.py

```diff
@@ -0,0 +1,204 @@
+
+'''
+Please pass in open ai api key and bill number as inputs. As of now this program can only work with 3 bills (H3069, H3121, H711)
```

Comment:
Below you actually operate on more than 3 bills, right?

Reply:
> Below you actually operate on more than 3 bills, right?

Yes, it does, but we need to update the CSV files with more testimonies of different bills, as of now I have collected testimonies of only 3 bills and stored them in the CSV.

Reply:
> Please also provide a little more detailed description of what the script does in this toplevel comment.

Sure thing